libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview.tsx

```diff
@@ -32,14 +46,12 @@ export class ModelOverview extends React.PureComponent<ModelOverviewProps> {
     }
 
     return (
-      <div className={classNames.page}>
-        <div className={classNames.infoWithText}>
-          <Text variant="medium">
-            {localization.Interpret.ModelPerformance.helperText}
-          </Text>
-        </div>
-        <OverallMetricChart />
-      </div>
+      <Stack tokens={{ padding: "16px 40px 10px 40px" }}>
```

Comment:
In the case if we will add more styling settings for this file in the future, shall we put styles into a separate .style file? 
But this could be done later since we only have one line now.

Reply:
Right, that's what I was thinking, too. If there are more in the future I'll definitely start one.